typespec-tests/src/main/java/com/_specs_/azure/clientgenerator/core/internal/models/PublicModel.java

```diff
@@ -16,7 +16,7 @@ public final class PublicModel {
      * The name property.
      */
     @Generated
-    @JsonProperty(value = "name", required = true)
+    @JsonProperty(value = "name")
```

Comment:
Technically we only need to remove the `required=true` in ctor. But I guess it is also fine here.